prow/cluster/jobs/all-periodics.yaml

```diff
@@ -209,7 +209,32 @@ periodics:
             value: release-1.4
         command:
           - entrypoint
-          - perf/benchmark/run_benchmark_job.sh
+          - perf/benchmark/run_istio_benchmark_job.sh
+
+- cron: "0 8 * * *" # starts every day at 08:00AM UTC
+  name: daily-linkerd-performance-benchmark
+  branches: master
+  decorate: true
+  timeout: 12h
```

Comment:
does this one take 12hr as well?

Reply:
it should be like around 3 hr actually. I just put it here in case we have more tests configurations.

Reply:
How does it become 3hrs? it looks like the test is just running at 6 different# of connections for 4min each shouldn't that be 24min?

Reply:
sorry, it should be  4min * 6 * 3= 72 min